<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawFormatDialog</class>
 <widget class="QDialog" name="RawFormatDialog">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>500</width><height>420</height></rect>
  </property>
  <property name="windowTitle">
   <string>RAW File Format Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing"><number>8</number></property>
   <property name="margin"><number>8</number></property>
   
   <!-- Row 1: Track Type, Tracks, Sides -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Track type:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboTrackType">
         <property name="minimumWidth"><number>120</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Number of Track:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinTracks">
         <property name="minimum"><number>1</number></property>
         <property name="maximum"><number>255</number></property>
         <property name="value"><number>80</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Number of side:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboSides">
         <item><property name="text"><string>1 Side</string></property></item>
         <item><property name="text"><string>2 Sides</string></property></item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSidesGrouped">
       <property name="text"><string>Tracks of a side
grouped in the file</string></property>
      </widget>
     </item>
    </layout>
   </item>
   
   <!-- Row 2: Bitrate, Sectors, Sector Size -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Bitrate:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBitrate">
         <property name="minimum"><number>100000</number></property>
         <property name="maximum"><number>1000000</number></property>
         <property name="value"><number>250000</number></property>
         <property name="singleStep"><number>1000</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Sector per track:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinSectors">
         <property name="minimum"><number>1</number></property>
         <property name="maximum"><number>255</number></property>
         <property name="value"><number>10</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Sector size:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboSectorSize">
         <item><property name="text"><string>128 Bytes</string></property></item>
         <item><property name="text"><string>256 Bytes</string></property></item>
         <item><property name="text"><string>512 Bytes</string></property></item>
         <item><property name="text"><string>1024 Bytes</string></property></item>
         <item><property name="text"><string>2048 Bytes</string></property></item>
         <item><property name="text"><string>4096 Bytes</string></property></item>
         <item><property name="text"><string>8192 Bytes</string></property></item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkReverseSide">
       <property name="text"><string>Reverse side</string></property>
      </widget>
     </item>
    </layout>
   </item>
   
   <!-- Row 3: RPM, Sector ID, Interleave -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>RPM:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboRPM">
         <item><property name="text"><string>300</string></property></item>
         <item><property name="text"><string>360</string></property></item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Sector ID start:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinSectorIdStart">
         <property name="minimum"><number>0</number></property>
         <property name="maximum"><number>255</number></property>
         <property name="value"><number>1</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkInterSideNumbering">
       <property name="text"><string>Inter side sector
numbering</string></property>
      </widget>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
    </layout>
   </item>
   
   <!-- Row 4: Interleave, Skew, Side Based -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Interleave:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinInterleave">
         <property name="minimum"><number>1</number></property>
         <property name="maximum"><number>99</number></property>
         <property name="value"><number>1</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Skew:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinSkew">
         <property name="minimum"><number>0</number></property>
         <property name="maximum"><number>99</number></property>
         <property name="value"><number>0</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSideBased">
       <property name="text"><string>Side based</string></property>
      </widget>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
    </layout>
   </item>
   
   <!-- Row 5: Total Sectors, Total Size, Format Value -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Total Sector:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editTotalSectors">
         <property name="readOnly"><bool>true</bool></property>
         <property name="text"><string>1600</string></property>
         <property name="maximumWidth"><number>80</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Total Size:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editTotalSize">
         <property name="readOnly"><bool>true</bool></property>
         <property name="text"><string>409600</string></property>
         <property name="maximumWidth"><number>80</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Format value:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editFormatValue">
         <property name="readOnly"><bool>true</bool></property>
         <property name="maximumWidth"><number>80</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
    </layout>
   </item>
   
   <!-- Row 6: GAP lengths -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>GAP3 length:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinGap3">
         <property name="minimum"><number>0</number></property>
         <property name="maximum"><number>255</number></property>
         <property name="value"><number>27</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkAutoGap3">
       <property name="text"><string>Auto GAP3</string></property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>PRE-GAP length:</string></property></widget></item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinPreGap">
         <property name="minimum"><number>0</number></property>
         <property name="maximum"><number>255</number></property>
         <property name="value"><number>0</number></property>
        </widget>
       </item>
      </layout>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
    </layout>
   </item>
   
   <!-- Row 7: Predefined Layout -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel"><property name="text"><string>Predefined Disk Layout:</string></property></widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLayout">
       <property name="minimumWidth"><number>200</number></property>
      </widget>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
    </layout>
   </item>
   
   <!-- Buttons -->
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnSaveConfig">
       <property name="text"><string>Save config</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadConfig">
       <property name="text"><string>Load config</string></property>
      </widget>
     </item>
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
     <item>
      <widget class="QPushButton" name="btnLoadRaw">
       <property name="text"><string>Load RAW file</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreateEmpty">
       <property name="text"><string>Create Empty
Floppy</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text"><string>Close</string></property>
      </widget>
     </item>
    </layout>
   </item>
   
   <!-- Info label -->
   <item>
    <widget class="QLabel" name="labelInfo">
     <property name="text"><string>To batch convert RAW files you can use the Batch Converter function and check the RAW files mode check box.</string></property>
     <property name="wordWrap"><bool>true</bool></property>
     <property name="styleSheet"><string>color: gray; font-size: 9pt;</string></property>
    </widget>
   </item>
   
  </layout>
 </widget>
</ui>
